charts/scalardl/values.yaml

```diff
@@ -173,6 +210,11 @@ ledger:
     # runAsNonRoot: true
     # runAsUser: 1000
 
+  # -- If you set "scalar.dl.ledger.proof.enabled=true" to ledger.ledgerProperties, you need to create volume that includes key file.
+  volumes: []
+  # -- If you set "scalar.dl.ledger.proof.enabled=true" to ledger.ledgerProperties, you need to mount key file to the path set in the "scalar.dl.ledger.proof.private_key_path".
+  volumeMounts: []
```

Comment:
[nits] In Grafana Chart, key names seem to be `extraVolumes` and `extraVolumeMounts`. It is indeed easier to understand that they are added.

- https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml#L457-L467

In addition, the help message mentions `scalar.dl.ledger.proof.enabled`, but the message needs to be updated each time volumes are used outside of the setting. I suggest changing the message to `Defines additional volume mounts` and supplementing the usage with documentation.

Reply:
Thank you for the suggestion!

> In Grafana Chart, key names seem to be `extraVolumes` and `extraVolumeMounts`. It is indeed easier to understand that they are added.

I tried to search similar settings and found some products use `extraVolumes`.

https://github.com/bitnami/charts/blob/master/bitnami/postgresql/values.yaml#L542-L547
https://github.com/elastic/helm-charts/blob/main/elasticsearch/values.yaml#L146-L153
https://github.com/hashicorp/consul-k8s/blob/main/charts/consul/values.yaml#L1224

It looks good to me since the name `extra` can imagine its use case.
So, I updated the name to `extraVolumes` and `extraVolumeMounts`.

> In addition, the help message mentions `scalar.dl.ledger.proof.enabled`, but the message needs to be updated each time volumes are used outside of the setting. I suggest changing the message to `Defines additional volume mounts` and supplementing the usage with documentation.

Indeed, it is better to make comment general. There is a possibility that we will have an another use case in the future.
I updated the comment based on the suggestion.
Also, I will explain how to use customized properties and how to mount key files in the document that I'm creating in the following PR.
https://github.com/scalar-labs/helm-charts/pull/104